backend/apps/volontulo/templates/emails/password_reset.txt

```diff
@@ -3,5 +3,5 @@
 Została złożona prośba resetu hasła dla użytkownika {{ email }}.
 
 Zmiany możesz dokonać za pomocą adresu:
-{{ ANGULAR_ROOT }}/password-reset/{{ uid }}/{{ token }}
+{{ ANGULAR_ROOT }}/forgot-password/{{ uid }}/{{ token }}
```

Comment:
I will not do the whole review here, but there are two different components (that are routable) here and You try to route here to wrong component.

See `PasswordResetComponent` and `PasswordResetConfirmComponent`